mkosi.profiles/devtools/mkosi.postinst

```diff
@@ -11,8 +22,26 @@ if [ -f "$BUILDROOT/etc/default/dropbear" ]; then
     # Remove -s, -w, -g flags from dropbear args
     sed -i '/^DROPBEAR_EXTRA_ARGS=/s/-[swg] \?//g' "$BUILDROOT/etc/default/dropbear"
 else
-    echo "PermitRootLogin yes" >> "$BUILDROOT/etc/ssh/sshd_config"
+    echo "PermitRootLogin yes"        >> "$BUILDROOT/etc/ssh/sshd_config"
     echo "PasswordAuthentication yes" >> "$BUILDROOT/etc/ssh/sshd_config"
     mkosi-chroot systemctl enable ssh.service
     mkosi-chroot systemctl unmask ssh.service ssh.socket
 fi
+
+# Create users and groups
```

Comment:
Can this be removed now? If you have custom scripts, you can add them to the devtools folder and use the git "skip worktree" feature to have git ignore them on your machine only